components/book-list/book-list.js

```diff
@@ -0,0 +1,25 @@
+import { LitElement, html } from "lit";
+import "../card/card";
+import styles from "./book-list.styles";
+
+class BookListComponent extends LitElement {
+  static get styles() {
+    return [styles];
+  }
+
+  render() {
+    return html`
+      <div class="container-book-list">
+        ${this.data.map(
+          (item) => html`
+            <div class="card">
```

Comment:
this div can be part of `card-component` 😬

Reply:
Corrections made